lib/delayed/backend/actions.rb

```diff
@@ -37,7 +48,9 @@ def find_available(worker_name, limit = 5, max_run_time = Worker.max_run_time)
             rescue Exception => e
               Delayed::Worker.logger.warn(e.message)
             end
-            return [Delayed::Backend::Ironmq::Job.new(message)] if message
+            if ready_to_run?(message)
```

Comment:
i fear some small amount of delayed jobs can totally block processing, DJ would grab same message over and over again never reaching non-delayed ones. 
Why not to use native "delay" MQ thing?